src/main/java/bio/terra/cloudfunctions/common/GsonWrapper.java

```diff
@@ -32,4 +32,8 @@ public static Gson getInstance() {
               .create();
     return _instance;
   }
```

Comment:
this singleton-creation method is not thread-safe. Multiple threads could hit `getInstance` simultaneously and run into each other. This is unlikely inside Cloud Functions but I think this should be thread-safe anyway. See https://www.geeksforgeeks.org/java-singleton-design-pattern-practices-examples/ for good examples. I think in this case it's probably safe to create the singleton at class initialization time.

Reply:
Updated `GsonWrapper` to return a thread-safe `Gson` instance.